.circleci/config.yml

```diff
@@ -22,6 +22,7 @@ install_dep: &install_dep
       command: |
         source activate mmf
         pip install --upgrade setuptools
+        pip install --progress-bar off -r requirements.txt
```

Comment:
I am somewhat not a fan of this as it creates confusion. `setup.py` should in itself be enough to install any dependencies.